playbook-k8s.yml

```diff
@@ -227,11 +237,14 @@
   - name: Install Pods
     become: yes
     become_user: vagrant
-    shell: kubectl apply -f kube-flannel.yml -f nginx-deployment.yml -f nginx-ingress.yml >> pods_setup.txt
```

Comment:
looks like you remove incorrect file, need to remove **nginx-ingress.yml** and leave **nginx-deployment.yml**
Because **nginx-ingress.yml** looks like incorrect ingress controller